src/css/profile/mobile/common/slider.less

```diff
@@ -6,18 +6,30 @@
 	flex-direction: column;
 	justify-content: space-around;
 
+	.ui-slider-bar::before {
+		content: '';
+		left: 0;
+		top: 0;
+		width: 100%;
+		height: 100%;
+		border-image: url(images/3_Controllers/tw_progress_bar_n.9.png) 5 fill;
+		opacity: 30%;
+		border-radius: 1.5 * @px_base;
+		position: absolute;
+
```

Comment:
You need two things:
```
border-image-width: 3px;
border-image-repeat: round;
```

Reply:
> You need two things:
> 
> ```
> border-image-width: 3px;
> border-image-repeat: round;
> ```

I checked again and I got a better result for:
` border-image-width: 1px;` without `  border-image-repeat: round;`